Resources/Maps/saltern.yml

```diff
@@ -23760,6 +23760,27 @@ entities:
   - pos: 14.5,21.5
     parent: 853
     type: Transform
+- uid: 2376
+  type: Bucket
+  components:
+  - pos: -19.613886,-0.09520197
+    parent: 853
+    type: Transform
+  - caps: Refillable, Drainable
```

Comment:
why wouldnt this just be on the bucket prototype itself?

Reply:
Uhhh idk?